stories/topics/con-gcp-upgrade-backup-process.adoc

```diff
@@ -0,0 +1,6 @@
+[id="con-gcp-upgrade-backup-process"]
+
+= Backing up before the upgrade
+
```

Comment:
The following procedures backup the deployment before the upgrade process begins.